src/client/dfuse/dfuse_main.c

```diff
@@ -332,11 +332,6 @@ main(int argc, char **argv)
 
 	D_GOTO(out_dfs, 0);
 
-out_cont:
-	if (dfuse_info->di_cont) {
-		dfs_umount(dfs->dfs_ns);
-		daos_cont_close(dfs->dfs_coh, NULL);
-	}
```

Comment:
But then where do the dfs_umount() and daos_cont_close() occur ?? In out_dfs: loop, right ? But then, why is it not also true for out_pool:/daos_pool_disconnect() ???

Reply:
I assume in the loop around 351 ?

Reply:
I am a little confused as well by this.

Reply:
I also thought about this, it turns out I've reworked this code in the caching patch and removed this block entirely there so it may be better to just go with those changes instead.